scrapers/data18.yml

```
@@ -0,0 +1,48 @@
+name: data18
+performerByURL:
+  - action: scrapeXPath
+    url: 
+      - data18.com/
+    scraper: performerScraper
+sceneByURL:
+  - action: scrapeXPath
+    url: 
+      - data18.com/content
+    scraper: sceneScraper
+xPathScrapers:
+  performerScraper:
+    performer:
+      Name: //h1
+      #Birthdate: 
+        #selector: //li[text()='Date of Birth: ']//span
+        #replace:
+          #- regex: ","
+            #with:
+        #parseDate: Jan 2, 2006
+      #Measurements: //li[text()='Measurements: ']//span
+  sceneScraper:
+    common:
+      $performer: //p[b[text()='Starring:']]/a
+      $studio: //p[b[text()='Site:']]/a
+    scene:
+      Title: //h1/text()
+      Date: 
+        selector: //span[contains(text(),'Release date:')]/a
+        replace: 
+          - regex: ","
+            with:
+        # reference date is: 2006/01/02
+        parseDate: January 2 2006
+      Details: //p[b[text()='Story:']]
```

this needs a bit of pp
```suggestion
      Details:
        selector: //p[b[text()='Story:']]
        postProcess:
          - replace:
              - regex: "^Story:"
                with:
              - regex: �
                with: "'"
```